deployer/templates/heapster.yaml

```diff
@@ -83,9 +82,9 @@ objects:
           - "--wrapper.username_file=/hawkular-account/hawkular-metrics.username"
           - "--wrapper.password_file=/hawkular-account/hawkular-metrics.password"
           - "--wrapper.allowed_users_file=/secrets/heapster.allowed-users"
-          - "--wrapper.endpoint_check=https://hawkular-metrics:443/hawkular/metrics/status"
+          - "--wrapper.endpoint_check=https://hawkular-metrics.openshift-infra.svc.cluster.local:443/hawkular/metrics/status"
```

Comment:
This is now hardcoding the project to 'openshift-infra'. You can either change this back to the way it was, or even better, get the namespace using the downward api and replace 'openshift-infra' with the actual namespace value